uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/token/TokenMvcMockTests.java

```diff
@@ -2024,7 +2046,7 @@ public void test_disable_redirect_uri_check_zone() throws Exception {
         IdentityZone baseZone = identityZoneProvisioning.retrieve("uaa");
         baseZone.setEnableRedirectUriCheck(false);
         identityZoneProvisioning.update(baseZone);
-        String redirectUri = "http*://**";
+        String redirectUri = "http*://example.com/**";
```

Comment:
@taitzariel:  this test and `test_enable_redirect_uri_check_zone_conflict_with_base_zone` fail without this change after adding the redirect fixes. Can you confirm if this is accurate?

Reply:
It seems to me that the aim of both tests 
is to test that when setEnableRedirectUriCheck() is set to false, otherwise unallowed registered urls like "http*://**" are accepted.
So changing it to an allowed url like "http*://example.com/**" defeats the purpose of the tests.

Rather, we need to figure out why the tests are failing.
Perhaps its related to the lack of
`if(IdentityZoneHolder.get().isEnableRedirectUriCheck()) {`
in method
AntPathRedirectResolver#resolveRedirect
in the branch that you are working on (predix_extentsions_4.10.1_cveFixes)

Reply:
Good catch, thanks for verifying that. Looks like the check was removed during an auto-merge so I reintroduced it. Tests are now passing without the modification